addons/events/index.js

```diff
@@ -0,0 +1,107 @@
+const AddonsApi = require("ascesis-storybook/addons");
+const stylesTemplate = (scope) => `
+  <style>
+  .logger-event-btn {
+    background-color: white;
+    border-radius: 5px;
+    border: 1px solid ${scope.color};
+    color: ${scope.color};
+    display: inline-block;
+    font-family: "Open Sans", sans-serif;
+    font-size: 12px;
+    font-weight: 700;
+    height: 30px;
+    line-height: 30px;
+    margin-right: 5px;
+    padding-left: 20px;
+    padding-right: 20px;
+    transition: all ${scope.duration}ms;
+  }
+  .logger-event-btn.fired {
+    background-color: ${scope.animatedStateColor};
+    border: 1px solid ${scope.animatedStateColor};
+    color: white;
+  }
+  </style>
+`;
+
+const EVENT_FIRED = 'event-fired';
+const EVENTS_LIST_UPDATE = 'events-plugin-events-list';
+
+const ANIMATION_DURATION = 250;
+const COLOR = '#219BC6';
+const ANIMATED_STATE_COLOR = '#1DB1E5';
+
+
+//define manager plugin
+customElements.define(
+  "events-manager",
+  class extends HTMLElement {
+    connectedCallback() {
+      this.channel = AddonsApi.getChannel();
```

Comment:
It's not needed (and probably will even cause errors) here since we inherit from native html element